tests/suites/os/tests/connectivity/index.js

```diff
@@ -53,12 +53,12 @@ module.exports = {
 
 							test.comment(`Attempting to connect to ${URL_TEST} over interface ${iface}`)
 							return this.worker.executeCommandInHostOS(
-								`ping -c 10 -i 0.002 -I ${iface} ${URL_TEST}`,
+								`curl -I -sS --interface ${iface} ${URL_TEST} | head -n 1`,
```

Comment:
```suggestion
								`curl -I -sS --interface ${iface} ${URL_TEST} | head -n 1`,
								`curl -I -sS -o /dev/null -w "%{http_code}" --interface ${iface} ${URL_TEST}`
```
cURL can be configured to only return the http code

Reply:
Yes, I decided against using this command to return only HTTP code since it felt like it was just grepping the HTTP code and checking the first line of the output felt like a more solid test. Not really it seems. Will swap this out.